Dockerfile

```
@@ -1,31 +1,40 @@
-FROM quay.io/opendatahub/text-generation-inference:stable-bafd218
-
-USER root
+ARG BASE_UBI_IMAGE_TAG=8.8-1067
+FROM registry.access.redhat.com/ubi8/ubi:${BASE_UBI_IMAGE_TAG} as base
 
 # Add grpc-ecosystem health probe
 ARG GRPC_HEALTH_PROBE_VERSION=v0.4.20
 
-WORKDIR /caikit
-COPY caikit /caikit
-
-RUN yum -y update && yum -y install git git-lfs && yum clean all && \
-    git lfs install && \
-    pip install 'micropipenv[toml]' && \
-    micropipenv install && \
-    rm -rf ~/.cache && \
-    mkdir -p /opt/models && \
-    adduser -g 0 -u 1001 caikit --home-dir /caikit && \
-    chown -R 1001:0 /caikit /opt/models && \
-    chmod -R g=u /caikit /opt/models
-
 # This is for the use-cases without kserve
 RUN curl -Lo /usr/local/bin/grpc_health_probe https://github.com/grpc-ecosystem/grpc-health-probe/releases/download/${GRPC_HEALTH_PROBE_VERSION}/grpc_health_probe-linux-amd64 && \
     chmod +x /usr/local/bin/grpc_health_probe
 
-USER 1001
+RUN dnf -y update && \
+    dnf -y install \
+        git git-lfs python39-pip python39-wheel && \
+    git lfs install && \
```

This seems to have gotten me past the issue have reported on the makefile
```suggestion
        git git-lfs python39-pip python39-wheel gcc python3-devel && \
```

---

The fact that `gcc` is required means that the wheels are not available and have to be built (using gcc). I think this is only required for arm (aarch64) targets, I'd rather avoid adding `gcc` to the base image.

---

I tend to agree here, we want the image to be very minimal. If there are python packages that we don't participate in and can't push arm wheels up to pypi, we may need a builder image for when we go to multiarch, but the release image shouldn't have gcc and dev tools per standard